core/templates/dev/head/pages/exploration_player/SupplementalCardDirective.js

```diff
@@ -0,0 +1,67 @@
+oppia.directive('supplementalCard', [function() {
+  return {
+    restrict: 'E',
+    scope: {
+      showNextCard: '&'
+    },
+    templateUrl: 'components/SupplementalCard',
+    controller: [
+    '$scope', 'oppiaPlayerService', 'UrlInterpolationService',
+    'playerPositionService', 'playerTranscriptService',
+    'ExplorationObjectFactory', 'windowDimensionsService', '$window',
+    function($scope, oppiaPlayerService, UrlInterpolationService,
+      playerPositionService, playerTranscriptService, explorationObjectFactory,
+      windowDimensionsService, $window) {
```

Comment:
Note that $window should be near the front -- we generally put all $-prefixed dependencies first in the list.